File: Habitica/src/release/java/com/habitrpg/android/habitica/ReleaseDeveloperModule.java
package com.habitrpg.android.habitica;


import com.habitrpg.android.habitica.modules.DeveloperModule;
import com.habitrpg.android.habitica.proxy.CrashlyticsProxyImpl;
import com.habitrpg.android.habitica.proxy.CrashlyticsProxy;

import android.content.Context;

//change debug proxy here by override methods
public class ReleaseDeveloperModule extends DeveloperModule {
    @Override protected CrashlyticsProxy provideCrashlyticsProxy(Context context) {
        return new CrashlyticsProxyImpl();
    }
}

File: Habitica/src/release/java/com/habitrpg/android/habitica/ReleaseDeveloperModule.kt
package com.habitrpg.android.habitica

import android.content.Context
import com.habitrpg.android.habitica.modules.DeveloperModule
import com.habitrpg.common.habitica.helpers.AnalyticsManager
import com.habitrpg.android.habitica.proxy.AnalyticsManagerImpl

class ReleaseDeveloperModule : DeveloperModule() {
    override fun provideAnalyticsManager(context: Context): AnalyticsManager {
        return AnalyticsManagerImpl(context)
    }
}
